public/src/components/trading_terminal/OrderBook/MarketDepth.tsx

```diff
@@ -80,13 +79,13 @@ export class MarketDepth extends React.Component<MarketDepthProps, {}> {
             }
             buyRows.push(
                 <tr key={i}>
-                    <td className="volume"><strong>{typeof(bidArray[i]) != "undefined" ? bidArray[i].volume : ''}</strong></td>
-                    <td className="price green"><strong>{typeof(bidArray[i]) != "undefined" ? bidArray[i].price : ''}</strong></td>
-                    <td className="volume"><strong>{typeof(askArray[i]) != "undefined" ? askArray[i].volume : ''}</strong></td>
+                    <td className="volume buy" id={"buy-volume"+i}><strong>{typeof(bidArray[i]) != "undefined" ? bidArray[i].volume : ''}</strong></td>
```

Comment:
It's almost never a good idea to concat using ```+```. Works most of the time but there'll always be some weird case. Most languages will provide some sort of templating method to concat integers and strings which is more readable and at the same time works as expected.